src/Experiment.jsx

```diff
@@ -2,30 +2,7 @@ import React, {Component} from "react";
 import PropTypes from 'prop-types';
 import CoreExperiment from "./CoreExperiment";
 import emitter from "./emitter";
-
```

Comment:
How does it help? Moving this from here to an independent file?

Reply:
@mahamshahid18 I had to reuse the same function in another file